tests/tools/oesign-engine/test-enclave/host/Makefile

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) Open Enclave SDK contributors.
```

Comment:
This file is completely unused. Please remove.

Reply:
removed already